app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MembersOnly</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+  <nav class="navbar navbar-dark bg-primary">
+      <a class="navbar-brand" href="#">
+        Members Only
+      </a>
```

Comment:
- **_[Optional] Change it to `<%= link_to 'Members Only', posts_path, class: "navbar-brand" %>`. It will improve the navigation of the webpage_**